src/Stripe.net/Services/Sources/SourceSourceOrderShippingOptions.cs

```diff
@@ -0,0 +1,14 @@
+namespace Stripe
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    public class SourceSourceOrderShippingOptions : ShippingOptions
+    {
+        [JsonProperty("carrier")]
+        public string Carrier { get; set; }
+
+        [JsonProperty("tracking_number")]
+        public string TrackingNumber { get; set; }
+    }
+}
```

Comment:
There's `ChargeShippingOptions` for this (the name is definitely not great but that's what we use in other places). Not sure what the best approach is here :(
Having a custom options one when the resource one is generic is weird. But maybe a problem from another time. Will defer to @ob-stripe on that one.